### ISSUE TITLE
DOI for bibliography entries

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#448]
Would it be possible to add DOI to the dropdown list of Additional Publication Info Entry types? I'm using General note at the moment, but would URL be the place to put this?

**Comment by ahankinson** [comment]
Yes, you can try putting it there. I think it will convert it into a clickable URL.

**Comment by ahankinson** [comment]
Turns out, several years later, that it does not in fact work. So I fixed it. It should now display both the url and the url accessed date. 

See: https://www.diamm.ac.uk/sources/1399/#/bibliography
